README.md

```diff
@@ -117,21 +117,36 @@ For vagrant documentation, please refer to [vagrantup.com](https://www.vagrantup
 
 This skeleton provides a `docker-compose.yml` for use with
 [docker-compose](https://docs.docker.com/compose/); it
-uses the `Dockerfile` provided as its base. Build and start the image using:
+uses the `Dockerfile` for a `laminas` container.  Build and start the image 
+using:
```

Comment:
I think it would be more clear if worded something like this:
```suggestion
uses provided `Dockerfile` to build docker image and run docker-compose service named `laminas`.
Build and start the image using:
```

Reply:
Applied and edited